broker/tests/unit/plugin/kubejobs/test_plugin.py

```diff
@@ -236,45 +324,40 @@ def test_wrong_request_body(self):
 
 class TestKubeJobsProvider(unittest.TestCase):
 
-    """
-    Set up KubeJobsProvider objects
-    """
-
     def setUp(self):
+        """
+        Set up KubeJobsProvider objects
+        """
         self.provider1 = KubeJobsProvider()
         self.provider2 = KubeJobsProvider()
 
     def tearDown(self):
```

Comment:
Is this really needed if it is not used?

Reply:
What you're talking about ?

Reply:
The empty tear_down method, it's not your code, but maybe we could remove the method, what do you think about it?